vcsutils/webhookparser/webhookparser.go

```diff
@@ -0,0 +1,40 @@
+package webhookparser
+
+import (
+	"net/http"
+
+	"github.com/jfrog/froggit-go/vcsutils"
+)
+
+const EventHeaderKey = "X-Event-Key"
+
+type WebhookInfo struct {
```

Comment:
Let's explain what this struct is used for. This would be helpful for the random reader.